groupbalancer.go

```diff
@@ -2,33 +2,41 @@ package kafka
 
 import "sort"
 
-// strategy encapsulates the client side rebalancing logic
-type strategy interface {
-	// ProtocolName of strategy
+// GroupMember describes a single participant in a consumer group.
+type GroupMember struct {
+	// ID is the unique ID for this member as taken from the JoinGroup response.
+	ID       string
+
+	// Topics is a list of topics that this member is consuming.
+	Topics   []string
+
+	// UserData contains any information that the GroupBalancer sent to the
+	// consumer group coordinator.
+	UserData []byte
+}
+
+// MemberGroupAssignments holds MemberID => topic => partitions
+type MemberGroupAssignments map[string]map[string][]int
```

Comment:
What's your take here on "MemberGroup" vs "GroupMember"?

Reply:
I like it.  Changed!